build.yaml

```diff
@@ -965,5 +1108,25 @@ job_type:
                    *build_repo_metadata,
                    'export HOME=/root',
                    *build_vagrant_tutorial_basebox,
-                   *clean_packages]
+                   *clean_packages
+                ]
+          }
+      timeout: 60
```

Comment:
increase the timeout for building the baseboxes as they're IO constrained on the slow SSDs